plenum/test/plugin/demo_plugin/test_request_digest.py

```diff
@@ -0,0 +1,32 @@
+import json
+
+from plenum.common.constants import CURRENT_PROTOCOL_VERSION, TXN_TYPE, DATA
+from plenum.common.request import Request
+from plenum.common.util import randomString
+
+from plenum.test.helper import sdk_random_request_objects, sdk_multisign_request_object, sdk_get_and_check_replies, \
+    sdk_send_signed_requests, sdk_gen_request, sdk_sign_and_submit_req_obj, sdk_get_reply
+from plenum.test.plugin.demo_plugin import dummy_field_length
+from plenum.test.plugin.demo_plugin.constants import PLACE_BID, AMOUNT, AUCTION_START
+
+
+def test_plugin_client_req_fields(txn_pool_node_set_post_creation, looper,
```

Comment:
Please add a test where two requests with the same payloads, but different plugin fields are sent. The second one is expected to be rejected.

Reply:
ok